agent/util-scripts/pbench-tool-data-sink

```diff
@@ -136,10 +139,8 @@ class BaseCollector:
 
 
 class PromCollector(BaseCollector):
-    allowed_tools = {"node-exporter": "9100", "dcgm": "8000"}
-
-    def __init__(self, benchmark_run_dir, host_tools_dict, logger):
-        super().__init__(benchmark_run_dir, host_tools_dict, logger)
+    def __init__(self, benchmark_run_dir, host_tools_dict, logger, tool_metadata):
+        super().__init__(benchmark_run_dir, host_tools_dict, logger, tool_metadata)
         self.volume = self.benchmark_run_dir + "/prom_vol"
```

Comment:
Can you convert this to use `Pathlib`, something like `self.volume = self.benchmark_run_dir / "prom_vol"`?

Reply:
And perhaps would could consider a directory hierarchy like:

```
 - <benchmark_run_dir>
   |- tools-<group>
      |- prometheus
      |- pcp
```